command/ca/provisionerbeta/add.go

```diff
@@ -404,7 +405,8 @@ func addAction(ctx *cli.Context) (err error) {
 			},
 			Enabled: !(ctx.IsSet("ssh") && !ctx.Bool("ssh")),
 		},
-		DisableRenewal: ctx.Bool("disable-renewal"),
+		DisableRenewal:        ctx.Bool("disable-renewal"),
```

Comment:
if we're using `disable-renewal` then shouldn't it be `allow-renewal-xxx`?

Reply:
Fixed with ff508dc